Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -1272,7 +1284,10 @@ void CSSParserImpl::consumeDeclaration(CSSParserTokenRange range, StyleRuleType
         }
     }
 
-    size_t propertiesCount = topContext().m_parsedProperties.size();
+    const size_t propertiesCount = topContext().m_parsedProperties.size();
+    auto isParsed = [&] {
+        return topContext().m_parsedProperties.size() != propertiesCount;
+    };
```

Comment:
Indeed, I've renamed those.

`propertiesCount` can be non zero though (this parses a single property, it will be non zero on the second property in the same scope/rule)/